docs/examples/bitops-deploy-heyemoji/test/terraform/bitops.before-deploy.d/create-tf-bucket.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+echo "Creating S3 bucket for Terraform state..."
+aws s3api create-bucket --bucket "$TF_STATE_BUCKET" --region "$AWS_DEFAULT_REGION" --create-bucket-configuration LocationConstraint=$AWS_DEFAULT_REGION || true
```

Comment:
Had to deal with some issues in another repo with this file. You can see the checks [here](https://github.com/bitovi/github-actions-deploy-docker-to-ec2/blob/main/operations/deployment/terraform/bitops.before-deploy.d/create-tf-state-bucket.sh)
Might be overkill ... but will prevent errors for users.
Shouldn't happen though.

Reply:
Good example!
Let me use that snippet.

Reply:
Actually, for the blog I think it's worth keeping the simple example considering the defaults we provide in the example, without caring much about the 63 characters logic limit.

It's not an user-facing app with GH Actions where more validation needed.

Reply:
Know it, thought of it, but just to keep it under the radar.

Reply:
Thanks for sharing!